docs/FOR VALIDATORS/Join a Testnet.md

```diff
@@ -0,0 +1,158 @@
+
+```text
+imversed validate-genesis
+```
+
+### Add Seed Nodes
+
+Your node needs to know how to find peers. You'll need to add healthy [seed nodes](https://docs.tendermint.com/v0.34/tendermint-core/using-tendermint.html#seed) to `$HOME/.imversed/config/config.toml`. The [`testnets`](https://github.com/imversed/testnets) repo contains links to some seed nodes.
+
+Edit the file located in `~/.imversed/config/config.toml` and the `seeds` to the following:
+
+```text
+#######################################################
+###           P2P Configuration Options             ###
+#######################################################
+[p2p]
+
+# ...
+
+# Comma separated list of seed nodes to connect to
+seeds = "<node-id>@<ip>:<p2p port>"
+```
+
+You can use the following code to get seeds from the repo and add it to your config:
+
+```text
+SEEDS=`curl -sL https://raw.githubusercontent.com/tharsis/testnets/main/imversed_5555558-1/seeds.txt | awk '{print $1}' | paste -s -d, -`
+sed -i.bak -e "s/^seeds =.*/seeds = \"$SEEDS\"/" ~/.imversed/config/config.toml
+```
+
+> For more information on seeds and peers, you can the Tendermint [P2P documentation](https://docs.tendermint.com/master/spec/p2p/peer.html).
+
+## Add Persistent Peers
+We can set the [`persistent_peers`](https://docs.tendermint.com/v0.34/tendermint-core/using-tendermint.html#persistent-peer) field in `~/.imversed/config/config.toml` to specify peers that your node will maintain persistent connections with. You can retrieve them from the list of available peers on the [`testnets`](https://github.com/imversed/testnets) repo.
+
+A list of available persistent peers is also available in the `#find-peers` channel in the [Imversed Discord](https://discord.gg/imversed). You can get a random 10 entries from the `peers.txt` file in the PEERS variable by running the following command:
+
+```text
+PEERS=`curl -sL https://raw.githubusercontent.com/tharsis/testnets/main/imversed_5555558-1/peers.txt | sort -R | head -n 10 | awk '{print $1}' | paste -s -d, -`
+```
+Use `sed` to include them into the configuration. You can also add them manually:
+
+```text
+sed -i.bak -e "s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" ~/.imversed/config/config.toml
+```
+
+## Run a Testnet Validator
+Claim your testnet tImversed on the [faucet](https://docs.imversed.com/developers/testnet/faucet.html) using your validator account address and submit your validator account address:
+
+> For more details on how to run your validator, follow `these` instructions.
+
+```text
+imversed tx staking create-validator \
+  --amount=1000000000000atimversed \
+  --pubkey=$(imversed tendermint show-validator) \
+  --moniker="ImversedWhale" \
+  --chain-id=<chain_id> \
+  --commission-rate="0.10" \
+  --commission-max-rate="0.20" \
+  --commission-max-change-rate="0.01" \
+  --min-self-delegation="1000000" \
+  --gas="auto" \
+  --gas-prices="0.025atimversed" \

```

Comment:
Change gas-prices=**"0.025atimversed"** to **"0.025aimv"**

Reply:
Updated...